packages/ag-charts-community/src/chart/layers.ts

```diff
@@ -7,6 +7,7 @@ export enum Layers {
     AXIS_ZINDEX = 20,
     SERIES_CROSSLINE_RANGE_ZINDEX = 30,
     SERIES_LAYER_ZINDEX = 500,
+    SERIES_ERRORBAR_ZINDEX = 600,
```

Comment:
It's unclear whether this is necessary or whether we could just use `SERIES_LAYER_ZINDEX` instead

Reply:
🤔 I'd need to look at the live scene-graph to work that out - let's come back to it.